code/game/objects/structures/crates_lockers/crates.dm

```diff
@@ -34,6 +34,17 @@
 	if(!src.can_open())
 		return FALSE
 
+	var/cc_packed = FALSE
+	for(var/obj/O in src)
+		if(O.density)
+			cc_packed = TRUE
+			break
+
+	if(cc_packed)
+		var/response = alert(usr, "This crate has been packed with bluespace compression, an item inside won't fit back inside. Are you sure you want to open it?","Bluespace Compression Warning", "No", "Yes")
+		if(response == "No")
+			return FALSE
```

Comment:
You can eliminate the need for the `cc_packed` var by doing something like this:
```dm
for(var/obj/O in src)
	if(O.density)
		var/response = alert(usr, "This crate has been packed with bluespace compression, an item inside won't fit back inside. Are you sure you want to open it?","Bluespace Compression Warning", "No", "Yes")
		if(response == "No")
			return FALSE
		break
```